src/SFA.DAS.FAA.Domain/Configuration/FindAnApprenticeship.cs

```diff
@@ -6,4 +6,5 @@ public class FindAnApprenticeship
     public string? RedisConnectionString { get; set; }
     public string? GoogleMapsApiKey { get; set; }
     public string? GoogleMapsId { get; set; }
+    public bool FeatureToggle { get; set; } = false;

```

Comment:
maybe name to AccountDeletionFeature if we are going down the feature toggle route?

Reply:
Yes, I Agree, Config name replaced to AccountDeletionFeature